src/core/server/server.ts

```diff
@@ -184,7 +187,7 @@ export class Server {
 
   public async setup() {
     this.log.debug('setting up server');
-    const setupTransaction = apm.startTransaction('server_setup', 'kibana_platform');
+    const setupTransaction = apm.startTransaction('server-setup', kibana-platform);
```

Comment:
nit: maybe move`kibana-platform` to constants?

Reply:
I asked not to use constants. This value is used in a single file. There is no benefit in declaring it as a constant.